components/fonts/font.rs

```diff
@@ -42,13 +43,14 @@ macro_rules! ot_tag {
     };
 }
 
-pub const GPOS: u32 = ot_tag!('G', 'P', 'O', 'S');
-pub const GSUB: u32 = ot_tag!('G', 'S', 'U', 'B');
-pub const KERN: u32 = ot_tag!('k', 'e', 'r', 'n');
-pub const SBIX: u32 = ot_tag!('s', 'b', 'i', 'x');
-pub const CBDT: u32 = ot_tag!('C', 'B', 'D', 'T');
-pub const COLR: u32 = ot_tag!('C', 'O', 'L', 'R');
-pub const BASE: u32 = ot_tag!('B', 'A', 'S', 'E');
+pub type Tag = u32;
```

Comment:
Nit: I think it's fine to make this alias here, but `Tag` is a bit overloaded in Servo. How about `OpenTypeTableTag`? I know it's a bit long, but I think it's a bit more descriptive.